scheduler/src/app.module.ts

```diff
@@ -7,16 +7,34 @@ import * as path from 'path';
 import { TpsPermitModule } from './modules/tps-permit/tps-permit.module';
 import { FeatureFlagsModule } from './modules/feature-flags/feature-flags.module';
 import { ScheduleModule } from '@nestjs/schedule';
-import { TypeormCustomLogger } from './logger/typeorm-logger.config';
-import { getTypeormLogLevel } from './helper/logger.helper';
+import { TypeormCustomLogger } from './common/logger/typeorm-logger.config';
+import { getTypeormLogLevel } from './common/helper/logger.helper';
 import { CacheModule } from '@nestjs/cache-manager';
 import { CgiSftpModule } from './modules/cgi-sftp/cgi-sftp.module';
+import { PermitModule } from './modules/permit/permit.module';
+import { ClsModule } from 'nestjs-cls';
+import { v4 as uuidv4 } from 'uuid';
 
 const envPath = path.resolve(process.cwd() + '/../');
 @Module({
   imports: [
     ScheduleModule.forRoot(),
     ConfigModule.forRoot({ envFilePath: `${envPath}/.env` }),
+    ClsModule.forRoot({
+      global: true,
+      middleware: {
+        // automatically mount the
+        // ClsMiddleware for all routes
+        mount: true,
+        generateId: true,
+        idGenerator: (req: Request) => {
```

Comment:
We only need this middleware config if scheduler is to receive http requests.

Reply:
okay

Reply:
removed